### ISSUE TITLE
2605.5.0 building kernel modules doesn't work

### ISSUE THREAD

**Issue by endzyme** [flatcar/Flatcar#195]
**Description**

Broken makefile in `build/` stops the ability to build modules which use `--with-linux-obj=/usr/lib64/modules/5.4.66-flatcar/build`. 

See `2512.5.0` in `/usr/lib64/modules/5.4.66-flatcar/build/Makefile` vs `2605.5.0`:

**2512.5.0**
```Makefile
# Automatically generated by ../source/scripts/mkmakefile: don't edit

VERSION = 4
PATCHLEVEL = 19

lastword = $(word $(words $(1)),$(1))
makedir := $(dir $(call lastword,$(MAKEFILE_LIST)))

ifeq ("$(origin V)", "command line")
VERBOSE := $(V)
endif
ifneq ($(VERBOSE),1)
Q := @
endif

MAKEARGS := -C ../source
MAKEARGS += O=$(if $(patsubst /%,,$(makedir)),$(CURDIR)/)$(patsubst %/,%,$(makedir))

MAKEFLAGS += --no-print-directory

.PHONY: __sub-make $(MAKECMDGOALS)

__sub-make:
	$(Q)$(MAKE) $(MAKEARGS) $(MAKECMDGOALS)

$(filter-out __sub-make, $(MAKECMDGOALS)): __sub-make
	@:
```

**2605.5.0**
```makefile
# Automatically generated by /build/amd64-usr/var/tmp/portage/sys-kernel/coreos-modules-5.4.66/work/coreos-modules-5.4.66/source/scripts/mkmakefile: don't edit
include /build/amd64-usr/var/tmp/portage/sys-kernel/coreos-modules-5.4.66/work/coreos-modules-5.4.66/source/Makefile
```

`/build/amd64-usr/var/tmp/portage/sys-kernel/coreos-modules-5.4.66/work/coreos-modules-5.4.66/source` doesn't exist..

I was able to get builds working by changing the Makefile to use `include /usr/lib64/modules/5.4.66-flatcar/source/Makefile`

**Impact**

Can't build kernel modules

**Environment and steps to reproduce**

1. **Set-up**: wget both the 2605.5.0 and the 2512.5.0 developer developer containers, and try to compile a module which uses `--with-linux-obj=...`

**Expected behavior**
Worked previously

**Comment by goochjj** [comment]
This is the solution I'm using
```
sed -i "s|/build/amd64-usr/var/tmp/portage/sys-kernel/coreos-modules-.*/work/coreos-modules-.*/source|/lib/modules/${KV}/source|g" /lib/modules/${KV}/build/Makefile
```
Where KV is the kernel version

**Comment by margamanterola** [comment]
This has been fixed with flatcar-linux/coreos-overlay#614, which will be included in the upcoming releases.

**Comment by endzyme** [comment]
Looks like it was released on https://www.flatcar-linux.org/releases/#release-2605.6.0
